src/Apps/Order/routes.tsx

```diff
@@ -57,7 +57,14 @@ export const routes: RouteConfig[] = [
             { order }
           )
           if (redirect !== null) {
-            throw new RedirectException(redirect)
+            if (process.env.NODE_ENV === "development") {
```

Comment:
As far as I can tell, sharify isn't set up correctly everywhere, while `process.env` is.

Reply:
From Storybooks it should be for most [commonly used things](https://github.com/artsy/reaction/blob/master/.storybook/webpack.config.js#L20-L56); and in Force it's something of the backbone of the app, so when a Reaction app is imported and then imports sharify it should work as expected.  

(If attempting to use `process.env.SOME_OTHER_VAR` it would break during the client-side render pass.)